### DISCUSSION TITLE
Ability to pull models has gone missing in v0.1.119

### DISCUSSION THREAD

**Discussion by sammcj** [open-webui/open-webui#1592] [General]
# Bug Report

## Description

**Bug Summary:**

After updating to v0.1.119 the ability to pull models seems to have disappeared!

**Steps to Reproduce:**

- Update to v0.1.119 or main
- Try to pull model that doesn't exist - not offered to pull anymore?
- Go to settings -> models - no option to pull a model anymore?

**Expected Behaviour:**

- To be offered to pull a model if it does not exist (from the models drop down)
- To be able to pull and manage models in settings -> models

**Actual Behaviour:**

- Not offered to pull
- No models in the settings anymore

## Environment

- **Operating System:** Linux/Docker
- **Browser (if applicable):** Firefox 125.0.1

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.

## Logs and Screenshots

<img width="602" alt="SCR-20240418-hton" src="https://github.com/open-webui/open-webui/assets/862951/f950596d-13e2-462a-8322-85e515d05ce5">

<img width="853" alt="SCR-20240418-htmp" src="https://github.com/open-webui/open-webui/assets/862951/328bbffc-9302-40df-a0d3-c80887c53841">

## Installation Method

- Docker

## Additional Information

N/A, but feel free to ask if you need any

**Comment by tjbck** [comment]
Seems like our webui couldn't connect to your Ollama instance, I'd suggest you check your network settings.

**Reply by sammcj** [reply]
Oh really! interesting, the rest of the app seemed to function normally, could select models and run inference without issue.

Perhaps there was an error that didn't surface in the UI, I'll keep any eye out!

**Reply by sammcj** [reply]
Turns out it's caused by this: https://github.com/open-webui/open-webui/issues/1800
